### PR TITLE
show which types of implementations work here

### PR DESCRIPTION
**Pull request by hardaker**
show which types of implementations work here

### CODE REVIEW

File: draft-iab-protocol-maintenance.md [hunk 1]
```diff
@@ -323,11 +323,12 @@ From observing widely deployed protocols, it appears there are two stable points
 on the spectrum between being strict versus permissive in the presence of
 protocol errors:
 
-* If implementations predominantly enforce strict compliance with
+* If continually updating implementations predominantly enforce strict compliance with
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:326]: I don't think that we want this change.  It dilutes the message too much.

**Review by martinthomson — CHANGES_REQUESTED** [review]
I think that this goes with #56.

File: draft-iab-protocol-maintenance.md (hunk 1 above)
Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md:326]: I agree. We already have this scope in the Applicability section.

File: draft-iab-protocol-maintenance.md [hunk 2]
```diff
@@ -323,11 +323,12 @@ From observing widely deployed protocols, it appears there are two stable points
...
   specifications, newer implementations will experience failures if they do not
   comply with protocol requirements. Newer implementations need to fix
   compliance issues in order to be successfully deployed. This ensures that most
-  deployments are compliant.
+  deployments are compliant over time.  This is especially true for
```
[the remark this answers is not in the document]
Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md:330]: +1

File: draft-iab-protocol-maintenance.md [hunk 3]
```diff
@@ -323,11 +323,12 @@ From observing widely deployed protocols, it appears there are two stable points
...
+  implementations that have expected rapid update cycles (for example web browsers).
```
[the remark this answers is not in the document]
Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md:331]: I'm not sure this adds much to the message in this document. We give guidance to implementations that can be updated, I don't think we need to say that it applies doubly so when the update cycles are twice as fast

**Review by DavidSchinazi — CHANGES_REQUESTED** [review]
I like the addition of "over time" but would rather avoid the other changes in this PR

**Pull request closed without merging**